pkg/cluster/root_app.go

```diff
@@ -141,5 +140,26 @@ func ConstructRootApp(
 		},
 		SyncOptions: []string{"Prune=true"},
 	}
+	fmt.Println(app.Spec.Source.Helm)
```

Comment:
A library function must not print to stdout. Is this for debugging? Remove.

Reply:
Yes, it is..